search_knn.go

```diff
@@ -194,6 +195,21 @@ func queryWithKNN(req *SearchRequest) (query.Query, error) {
 	return req.Query, nil
 }
 
+func setStoreForKNN(req *SearchRequest, coll *collector.TopNCollector, size int) {
+	if len(req.KNN) > 0 {
+		kArray := make([]int64, 1+len(req.KNN))
```

Comment:
i think in the codebase the convention is +1, rather than 1+. Just to keep things similar, could you please change it to +1?